app/src/main/java/protect/card_locker/ImportURIHelper.java

```diff
@@ -46,8 +46,15 @@ public ImportURIHelper(Context context) {
     }
 
     private boolean isImportUri(Uri uri) {
+        String uriPath = uri.getPath();
+        if (uriPath.endsWith("/")) {
+            // Remove the trailing slash if it exists
+            // some browsers such as firefox will add it
+            uriPath = uriPath.replaceAll("/$", "");
+        }
+
```

Comment:
```suggestion
        // Remove trailing slash added by some browsers (if it exists)
        String uriPath = uri.getPath().replaceAll("/$", "");
```

Reply:
Was thinking we could avoid compiling the regex and what-not, but it's probably a useless optimization :man_shrugging:

Reply:
Fair enough. I think less code is better than an optimisation that's unlikely to matter much. @TheLastProject has the last word of course :)